src/ray/common/asio/instrumented_io_context.cc

```diff
@@ -18,7 +18,15 @@
 #include <iomanip>
 #include <iostream>
 #include <utility>
-
+#include "ray/stats/metric.h"
+
+DEFINE_stats(io_pool_req_num, "IO pool request num", {"Method"}, {}, ray::stats::GAUGE);
+DEFINE_stats(io_pool_req_execution_time_us, "IO pool execution time", {"Method"}, {},
```

Comment:
Is there a stat that tracks overall latency (Queue + execution)?

Reply:
I think (not sure) that Prometheus can do adding or two metrics, so later we just build a dashboard with that (queue + execution).  @rkooo567  do you have some idea about this?

Reply:
Yeah Prometheus should be able to do that I believe